dj_cqrs/utils.py

```diff
@@ -3,67 +3,48 @@
 import logging
 from datetime import timedelta
 
-from dj_cqrs.constants import DEFAULT_CQRS_MESSAGE_TTL, DEFAULT_DELAY_QUEUE_MAX_SIZE
-
 from django.conf import settings
 from django.utils import timezone
 
 
 logger = logging.getLogger('django-cqrs')
 
 
-def get_expires_datetime():
+def get_message_expiration_dt():
     """Calculates when message should expire.
 
-    :return: datetime instance, None if infinite
-    :rtype: datetime.datetime
+    :return: Expiration datetime or None if infinite
+    :rtype: datetime.datetime or None
     """
-    master_settings = settings.CQRS.get('master', {})
-    if 'CQRS_MESSAGE_TTL' in master_settings and master_settings['CQRS_MESSAGE_TTL'] is None:
+    message_ttl = settings.CQRS['master']['CQRS_MESSAGE_TTL']
+    if message_ttl is None:
         # Infinite
         return
 
-    min_message_ttl = 1
-    message_ttl = master_settings.get('CQRS_MESSAGE_TTL', DEFAULT_CQRS_MESSAGE_TTL)
-    if not isinstance(message_ttl, int) or message_ttl < min_message_ttl:
-        logger.warning(
-            "Settings CQRS_MESSAGE_TTL=%s is invalid, using default %s.",
-            message_ttl, DEFAULT_CQRS_MESSAGE_TTL,
-        )
-        message_ttl = DEFAULT_CQRS_MESSAGE_TTL
-
     return timezone.now() + timedelta(seconds=message_ttl)
 
 
 def get_delay_queue_max_size():
-    """Returns delay queue "waiting" messages number.
+    """Returns max allowed number of "waiting" messages in the delay queue.
 
-    :return: integer instance, None if infinite
+    :return: Positive integer number or None if infinite
     :rtype: int
     """
-    replica_settings = settings.CQRS.get('replica', {})
-    max_size = DEFAULT_DELAY_QUEUE_MAX_SIZE
-    if 'delay_queue_max_size' in replica_settings:
-        max_size = replica_settings['delay_queue_max_size']
+    if 'replica' not in settings.CQRS:
```

Comment:
Now consumer will always have a replica settings with CQRS_DELAY_QUEUE_MAX_SIZE.

Just thoughts - now we could use settings.CQRS['replica']['CQRS_DELAY_QUEUE_MAX_SIZE'] directly.
But for me header comments are very useful and i preferred to stay with 1 line function.

Reply:
I see

Reply:
Can you comment why we should check replica settings here? Looks like they will be always there for replica workers.